tutorials/plot_sensors_time_frequency.py

```diff
@@ -115,6 +115,15 @@
 plt.show()
 
 ###############################################################################
+# You can also create a joint plot that contains both the aggregated TFR
+# across channels and topomaps at specific times and frequencies.
+
+power.plot_joint(picks=range(82, 84), baseline=(-0.5, 0), mode='logratio',
+                 title='auto', timefreqs=[(1, 10), (1, 19)],
+                 timefreqs_average=[(1, 4), (1, 6)])
```

Comment:
For the implementation, I suggest something like

```
if not isinstance(timefreqs, dict):
    timefreqs = {k:(1, 1) for k in timefreqs}

```

So you always have a dict.
At least that's how I do it in `plot_compare_evokeds` and I think it works fine.